app/views/categories/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1 class="new-form">New category</h1>
+
+<%= render "form", category: @category %>
+
+<br>
+
+<div class="back">
+  <%= link_to "Back to categories", categories_path %>
+</div>
```

Comment:
- [ ] Please place the "Back" button (<) at the top navigation of the page just as in the Behance design shown below 👇 
### Your Design
![image](https://github.com/muhammedb8826/budget-app/assets/46839548/bb5b5da8-4174-4e70-a641-3d84d8c13a56)
### Design Guideline
![image](https://github.com/muhammedb8826/budget-app/assets/46839548/b8dfec98-2e56-4003-9b2c-0f9903dce92e)